Zend/zend_language_scanner_defs.h

```diff
@@ -1,4 +1,4 @@
-/* Generated by re2c 0.16 */
+/* Generated by re2c 1.0.3 */
```

Comment:
Is this version mismatch okay?

Reply:
Hmm, I presume the re2c version we are suppose to be using is 0.16? This version [is ~2.5 years old](http://re2c.org/news/changelog/changelog.html) and doesn't seem to be receiving bug fixes or new features. I wonder if it is time to upgrade this to 1.x?

Reply:
The used re2c version changes regularly, it's just a matter of who touched the file last and which version they happened to have installed.